### PR TITLE
Fix: Fix the query sharing page

### PR DESCRIPTION
**Pull request by tonilastre**
### Description

Fixes applied to the Query Sharing page from the issue: https://github.com/memgraph/documentation/issues/889

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
Issue https://github.com/memgraph/documentation/issues/889

Closes:

Closes https://github.com/memgraph/documentation/issues/889

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

File: pages/data-visualization/user-manual/query-sharing.mdx [hunk 1]
```diff
@@ -33,6 +33,167 @@ started in a Docker environment.
...
+  -e STORAGE_MG_HOST=127.0.0.1 \
+  -e STORAGE_MG_PORT=7867 \
+  -e STORAGE_MG_IS_ENCRYPTED=true \
+  -e STORAGE_MG_USERNAME=lab \
+  -e STORAGE_MG_PASSWORD=lab \
+  memgraph/lab
+```
+
+If there is no username or password and Memgraph is accepting
+non-SSL connections, just omit the optional environment variables:
```
Comment (hal-eisen-MG) [inline pages/data-visualization/user-manual/query-sharing.mdx:106]: Never use the word "just"

Comment (tonilastre) [inline pages/data-visualization/user-manual/query-sharing.mdx:106]:
```suggestion
non-SSL connections, omit the optional environment variables:
```

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Comment by tonilastre** [comment]
@katarinasupe can you also check if this solves all the issues you mentioned in the Issue?

File: pages/data-visualization/user-manual/query-sharing.mdx [hunk 2]
```diff
@@ -33,6 +33,167 @@ started in a Docker environment.
 
 </Callout>
 
+### How to set it up
+
+To enable the Query Sharing feature in the Lab, you need to set up
+the remote storage (Memgraph instance) that the Lab will connect to
+and store query-sharing data.
+
+Query sharing requires a [valid Memgraph Enterprise license](/database-management/enabling-memgraph-enterprise).
```
Comment (katarinasupe) [inline pages/data-visualization/user-manual/query-sharing.mdx:42]:
```suggestion
Query sharing requires a [valid Memgraph Enterprise license](/database-management/enabling-memgraph-enterprise)
```

File: pages/data-visualization/user-manual/query-sharing.mdx [hunk 3]
```diff
@@ -33,6 +33,167 @@ started in a Docker environment.
...
+docker run \
+  -p 3000:3000 \
+  -e ENTERPRISE_LICENSE_ORG_NAME="My company" \
+  -e ENTERPRISE_LICENSE_KEY="key..." \
+  memgraph/lab
+```
+
+The enterprise license will be checked once you set up the
+remote storage. Remote storage should be a new Memgraph instance
+or a new Memgraph database in a multi-tenant environment.
```
Comment (katarinasupe) [inline pages/data-visualization/user-manual/query-sharing.mdx:63]: Suggestion - link to sections below.

File: pages/data-visualization/user-manual/query-sharing.mdx [hunk 4]
```diff
@@ -33,6 +33,167 @@ started in a Docker environment.
...
+of the Memgraph instance:
+
+| Variable                   | Description                                       | Type       |
+|----------------------------|---------------------------------------------------|------------|
+| `STORAGE_MG_IS_ENCRYPTED`  | Memgraph SSL on/off for the Lab remote storage    | `boolean`  |
+| `STORAGE_MG_USERNAME`      | Memgraph username for the Lab remote storage      | `string`   |
+| `STORAGE_MG_PASSWORD`      | Memgraph password for the Lab remote storage      | `string`   |
+
+Let's say that you already created a new database called `lab`
+in your enterprise Memgraph instance. For better security, it
```
Comment (katarinasupe) [inline pages/data-visualization/user-manual/query-sharing.mdx:154]:
```suggestion
Let's say that you already created a new database called `lab`
in your enterprise Memgraph instance for remote storage. For better security, it
```

**Review by katarinasupe — CHANGES_REQUESTED** [review]
- Fix title - it should be h1 (check GraphChat). 
- Use the same capitalization case across page (remove title case from the titles and use sentence-style capitalization; you can use title case for the feature name to keep it the same across the page)

Other than that, I love the new flow and all is clear now! :D

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**